src/github.com/getlantern/analytics/analytics.go

```diff
@@ -68,8 +68,21 @@ type Payload struct {
 	Event *Event
 }
 
+func Configure(trackingId string, version string, proxyAddr string) {
+	var err error
+	go func() {
+		httpClient, err = util.HTTPClient("", proxyAddr)
+		if err != nil {
+			log.Errorf("Could not create HTTP client via %s: %s", proxyAddr, err)
+			return
+		}
+		// Store new session info whenever client proxy is ready
+		sessionEvent(version, trackingId)
```

Comment:
We shouldn't be doing any analytics setup if auto-report is disabled. I moved the ``Configure()`` inside the autoReport check in ``flashlight/analytics.go``. This wouldn't matter to FT where we have no such setting.